fip-4.md

```diff
@@ -0,0 +1,94 @@
+---
+fip: 4
```

Comment:
Let's keep each FIPs in their own PRs.

Reply:
ill split this into independent PRs from distinct branches...